experimental/conf.py

```diff
@@ -135,9 +138,17 @@
 # POPULATE LINKS TO OTHER VERSIONS
 html_context['versions'] = list()
 
-tags = sorted(repo.tags, key=lambda t: t.commit.committed_datetime)
-versions = [branch.name for branch in tags]
-versions = versions[len(versions)-1:]
+tags = sorted(repo.tags, key=lambda t: t.commit.committed_datetime, reverse=True)
+versions = list()
+currentMinor = 0
+for tag in tags:
+    current = Version(tag.name)
+    if current.minor != currentMinor and current.minor >= 8:
+        versions.append(tag.name)
+        currentMinor = current.minor
```

Comment:
I think this would work perfectly for pre-v1.0.0 versions, but if we have versions such as `['v1.0.0', 'v0.9.11', 'v0.9.10', 'v0.9.8', 'v0.8.11']`, this might only generate `['v0.9.11', 'v0.8.11']`?